### PR TITLE
add Stefan–Boltzmann constant

### PR DESCRIPTION
**Pull request by shivakrishna9**

### CODE REVIEW

File: sympy/physics/units.py [hunk 1]
```diff
@@ -260,6 +260,7 @@ def free_symbols(self):
 avogadro_number = Rational('6.022140857') * 10**23
 avogadro = avogadro_constant = avogadro_number / mol
 boltzmann = Rational('1.38064852') * ten**-23 * J / K
+stefan_boltzman = Rational('5.670367')*ten**-8*W*m**-2*K**-4
```
Comment (smichr) [inline sympy/physics/units.py]: `man`->`mann`

Comment (smichr) [inline sympy/physics/units.py]: what about `stefan = stefan_boltzmann = ...`?

Comment (shivakrishna9) [inline sympy/physics/units.py]: It should be boltzmann. I didnot understand your second comment. Please clarify @smichr .

Comment (smichr) [inline sympy/physics/units.py]: Just letting the constant be called `stefan` or `stefan_boltzmann` which, the wiki said, was another name that it goes by.

**Comment by smichr** [comment]
see also #12450 
perhaps this should wait for #12246 , too

**Comment by smichr** [comment]
OK, the unification of units is complete. You can make your mod on top of the new master and repush here with the `-f` option. I made some comments about how to do that easily (if you don't already know)  [here](https://github.com/sympy/sympy/pull/12424#issuecomment-290398966).

**Comment by gxyd** [comment]
@shivakrishna9 can you please merge master branch into your PR? and also address the requested changes.

**Comment by shivakrishna9** [comment]
@gxyd Sure, onto it. Thanks.

**Comment by gxyd** [comment]
Hi @shivakrishna9 you haven't merged master branch.

**Comment by gxyd** [comment]
Hi @shivakrishna9 do you need any help with git?

**Pull request closed without merging**